docs/data_driven_testing.adoc

```diff
@@ -1,4 +1,4 @@
-= Data Driven Testing
+= [[data-driven-testing]]Data Driven Testing
```

Comment:
Please use the following format, alternatively you can use the auto links feature `_data_driven_testing`

```
[[data-driven-testing]]
= Data Driven Testing
```

Reply:
Yeah, just got told it should not be an inline anchor, Asciidoctor docs were a bit too misleading there.
What do you mean with auto links feature?
You mean the auto-generated anchors / ids?
Besides that I'm not a fan of linking to auto-generated ids as they can change, for example if another section with the same content is added, there is no anchor generated in the split output.

Reply:
In the comment below you say, that it warns about wrong ids, so shouldn't this fix the problem with changing ids? I'm split on this issue, on the one hand it is redundant information and clutters the adoc on the other hand it provides more resilience to changes.

Reply:
It would not.

Assuming you add some other section that is also called `Data Driven Testing` as some sub headline in another chapter and it comes before this headline, suddenly the other section is linked to an no check in the world besides manually clicking all links can find that.

Besides that, it is not really an option to use the auto-generated id in this case, because the top-level section of a document does **not** get an id.
So in the multi-document variant, the section has no id or anchor.
Only in the one-page variant it gets an auto-generated id.
I could at most explicitly assign the id that is also generated implicitly in the one-page variant.
Otherwise the link will link to an unknown id in the multi-page variant.